js/add-books.js

```diff
@@ -0,0 +1,49 @@
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const add = document.querySelector('.submit');
+const allBooks = document.querySelector('#all-books');
+const error = document.querySelector('.error');
+
+const bookStore = JSON.parse(localStorage.getItem('books')) || [];
+
+function showBooks() {
+  for (let i = 0; i < bookStore.length; i += 1) {
+    allBooks.innerHTML += `
+    <article class="books">
+      <p>${bookStore[i].title}</p>
+      <p>${bookStore[i].author}</p>
+      <button class="remove" onclick="removeBook(${i})">Remove</button>
+      <hr>
+    </article>
+`;
+  }
+}
+
+function addBooks(event) {
+  event.preventDefault();
+  if (title.value && author.value) {
+    const newBooks = {
+      title: title.value,
+      author: author.value,
+    };
+    bookStore.push(newBooks);
+    localStorage.setItem('books', JSON.stringify(bookStore));
+    showBooks();
+    title.value = '';
+    author.value = '';
+    error.innerHTML = '';
+  } else {
+    error.innerHTML = 'Please enter a book';
+  }
+}
+
+function removeBook(index) {
+  bookStore.splice(index, 1);
+  localStorage.setItem('books', JSON.stringify(bookStore));
+  showBooks();
+}
```

Comment:
Great job team 🤝

- [ ] When the delete button is clicked to remove the book from the collection, it is removed from the local storage but they aren't removed from the UI. Kindly ensure the book removed from the local storage is also removed from the UI.
![deleted](https://github.com/jecode93/awesome-book/assets/63917725/e6c96c14-27e7-4db7-8ebb-c57130de30e3)

Reply:
@1GORDON we have the same problem here, to see the changes we have to refresh the page..

How do you think we could solve this, please..?